src/collectd.conf.pod

```diff
@@ -7503,7 +7504,8 @@ the timestamp as the score. Retrieving a date range can then be done using the
 C<ZRANGEBYSCORE> I<Redis> command. Additionnally, all the identifiers of these
 I<Sorted Sets> are kept in a I<Set> called C<collectd/values> (or
 C<${prefix}/values> if the B<Prefix> option was specified) and can be retrieved
-using the C<SMEMBERS> I<Redis> command. See
+using the C<SMEMBERS> I<Redis> command. You can specify the database to use 
+with the Database parameter (default is 0). See
 L<http://redis.io/commands#sorted_set> and L<http://redis.io/commands#set> for
 details.
 
```

Comment:
Can you please add a more detailed description below? Something like:

```
=item B<Database> I<Index>

Selects the database … Defaults to C<0>.
```

See the *perlpod(1)* manpage for a description of the syntax.